osu.Game/Screens/Play/PlayerLoader.cs

```diff
@@ -51,7 +51,7 @@ public class PlayerLoader : ScreenWithBeatmapBackground
         // We show the previous screen status
         protected override UserActivity? InitialActivity => null;
 
-        protected override bool PlayResumeSound => false;
+        protected override bool PlayExitSound => false;
```

Comment:
I understand why the others are not playing the exit sound, but this seems like something we potentially don't want? Hitting escape while in the loading sequence now feels very lacklustre where it used to feel quite good.

Reply:
Hrm, I did this as to not regress #4678, but as you say it should probably play in those other cases.

Reply:
I'll take a look at this unless you've already started to.

Reply:
Go for it.

Reply:
I think we also want the sound to play on exiting the editor still?

https://user-images.githubusercontent.com/191335/173774761-c3efb3b5-ee16-46ed-99ff-3b7087d9590d.mp4

Reply:
Also screens like `IntroTriangles` are now playing an exit sound, which I presume is not the intention.

Reply:
Huh, I didn't realise you could exit the editor without invoking the 'save changes' dialog. I agree it sounds awkward there without the back sound.

My intention was just to not have the 'back' sound playing over the dialog samples when exiting the editor via this flow:

https://user-images.githubusercontent.com/272140/173777088-cab4fbee-e9f9-4ae6-8414-8a8c1e869470.mov

Reply:
Yeah. If the Editor save dialog appears (and as a result, one of the button samples plays), then the 'back' sample shouldn't also play. The other scenarios were oversights on my behalf and should behave as before.